knative-operator/cmd/manager/main.go

```diff
@@ -166,8 +173,36 @@ func setupServerlesOperatorMonitoring(cfg *rest.Config) error {
 		return fmt.Errorf("failed to setup monitoring resources: %w", err)
 	}
 
-	if err := health.InstallHealthDashboard(cl); err != nil {
-		return fmt.Errorf("failed to setup the Knative Health Status Dashboard: %w", err)
+	apiExtensionClient, err := apiextension.NewForConfig(cfg)
+	if err != nil {
+		return err
+	}
+
+	var installedDashboard bool
+	if _, err = apiExtensionClient.ApiextensionsV1().CustomResourceDefinitions().Get(context.Background(), consoleclidownload.CLIDownloadCRDName, metav1.GetOptions{}); err == nil {
+		consoleclidownload.SetConsoleCRDInstalled(consoleclidownload.CLIDownloadCRDName)
+		if err := health.InstallHealthDashboard(cl); err != nil {
+			return fmt.Errorf("failed to setup the Knative Health Status Dashboard: %w", err)
+		}
+		installedDashboard = true
+	} else {
+		if !apierrors.IsNotFound(err) {
+			return fmt.Errorf("failed to fetch ConsoleCLIDownload CRDs: %w", err)
```

Comment:
We shouldn't return an error, if CRD does not exist. That is valid, and expected case. Instead, we should log it.

Otherwise, we will see false errors in logs.

Reply:
right ok will fix thanks.